### PR TITLE
Added EventListener to shift pixels by arrow keys and keyboard shortcuts

### PR DESCRIPTION
**Pull request by RegularRolfe**
When `ctrl` and any arrow keys are pressed, the appropriate `shift` functions are called. More keys can be added into the `switch` statement. 

To look at additional keycodes, I reccommend using [this website](https://keycode.info/).

For other key combinations, use the altKey, metaKey or shiftKey properties.

### CODE REVIEW

File: scripts/editor/functions.js [hunk 1]
```diff
@@ -354,3 +354,39 @@ function copyToClipboard () {
...
+      case "ArrowDown":   shiftDown();
+        break;
+      case "ArrowLeft":   shiftLeft();
+        break;
+      case "ArrowRight":  shiftRight();
+        break;
+
+      /* 
+      ! These following keys may have conflicts with the browsers' hotkeys 
+      ! You can (should) change it to whatever feels best
```
Comment (alestiago) [inline scripts/editor/functions.js]: Should we instead change it to Alt? Or any other equivalent that doesn't toggle browsers hotkeys?

Comment (RegularRolfe) [reply scripts/editor/functions.js]: That would be challenging as it also has conflicts with `Ctrl+Shift`, `Ctrl+Alt`, and `Alt+Shift` combinations. I am against using more than two keys as that defeats the purpose. 

Will look into this further, maybe there's something that will block browser/OS keyboard shortcuts.

Comment (RegularRolfe) [reply scripts/editor/functions.js]: Okay, browser hotkeys are disabled with `event.preventDefault();`

**Comment by alestiago** [comment]
The solution to overwrite browser keyboard causes an issue when inputting  the name of the character name.

**Comment by RegularRolfe** [comment]
>inputting the name of the character name.

Sorry I don't know what you mean by this. Could you describe it further?

**Comment by alestiago** [comment]
nameofchar
Try renaming the character, it will conflict. In the meantime I have disabled the overwrite and changed from ctrl key to shift key. Will be nice though if you manage to use the ctrl key.

**Pull request merged**